radixdlt-core/radixdlt/src/test/java/com/radixdlt/client/AccountAddressTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * (C) Copyright 2021 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.client;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.HashBiMap;
+import com.radixdlt.crypto.ECKeyPair;
+import com.radixdlt.identifiers.REAddr;
+import com.radixdlt.serialization.DeserializeException;
+import com.radixdlt.utils.Bytes;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class AccountAddressTest {
+	private final BiMap<String, String> privateKeyToAccountAddress = HashBiMap.create(
+		Map.of(
+			"00", "brx1qsps28kdn4epn0c9ej2rcmwfz5a4jdhq2ez03x7h6jefvr4fnwnrtqqjqllv9",
+			"deadbeef", "brx1qspsel805pa0nhtdhemshp7hm0wjcvd60a8ulre6zxtd2qh3x4smq3sraak9a",
+			"deadbeefdeadbeef", "brx1qsp7gnv7g60plkk9lgskjghdlevyve6rtrzggk7x3fwmp4yfyjza7gcumgm9f",
+			"bead", "brx1qsppw0z477r695m9f9qjs3nj2vmdkd3rg4mfx7tf5v0gasrhz32jefqwxg7ul",
+			"aaaaaaaaaaaaaaaa", "brx1qspqsfad7e5k2k9agq74g40al0j9cllv7w0ylatvhy7m64wyrwymy5g7md96s"
+		)
+	);
+
+	private final BiMap<String, String> reAddrToAccountAddress = HashBiMap.create(
+		Map.of(
+			"03" + "03".repeat(26), "brx1qvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvpsxqcrqvpsytf8zx",
```

Comment:
uhm, this is 26 bytes only, this is the recipe for RRIs, should not work with AccountAddress, right? 

If it does, we allow for AccountAddresses NOT containing PublicKey and this breaks stuff, e.g. encryption wont work. I thought we where going to enforce the compressed public key as data for now, and ONLY allow that, so that we on JS side ALWAYS can retrieve the public key from an AccountAddress.

So hmm... I would be happy to see an update to impl that enforces 33 bytes data (compressed public key) and.